src/components/accordion/accordion.twig

```diff
@@ -0,0 +1,28 @@
+<section class="accordion-section">
+  <div class="accordion" data-action="is-accordion" role="tablist">
+    <div class="js-accordion__panel">
+      <div role="heading">
```

Comment:
can you swap out the level-less ARIA headings for actual heading elements instead?  The practices guide goes far in the direction of using ARIA for all the things, because its purpose is to show how to use ARIA.  but ideally the correct HTML elements should be used instead of ARIA.